src/components/performance.cjsx

```diff
@@ -14,8 +14,27 @@ Performance = React.createClass
   propTypes:
     courseId: React.PropTypes.string.isRequired
 
+  getInitialState: ->
+    sortOrder: 'asc'
+    sortIndex: 0
+    isNameSort: true
+
+  sortClick: (event) ->
+    if not _.contains(event.target.classList, 'student-name')
+      @setState({isNameSort: false})
+    else
+      @setState({isNameSort: true})
+    headers = event.target.parentNode.querySelectorAll('th')
+    for header in headers
+      header.classList.remove('asc', 'desc')
```

Comment:
I'm not sure how I would isolate the clicked element from render; it seems more straightforward to do the class manipulation this way.